sky/shell/platform/ios/framework/Source/accessibility_bridge.h

```diff
@@ -39,12 +36,7 @@ class AccessibilityBridge;
  * The parent of this node in the node tree. Will be nil for the root node and
  * during transient state changes.
  */
-@property(nonatomic, readonly) AccessibilityNode* parent;
-
-/**
- * This node's children in the node tree.
- */
-@property(nonatomic, readonly) NSArray<AccessibilityNode*>* children;
```

Comment:
Changing to a `std::vector` and `std::unordered_map` means you'll have to manually manage the ref counts of the nodes (previously `NSArray` and `NSMutableDictionary` were doing that for you).